### ISSUE TITLE
Chipstuete aufraeumen

### ISSUE THREAD

**Issue by grindhold** [openlab-aux/orgafoo#157]
Im Lab wurde von einem Mitglied diese Chipstuete gefunden

![Chipstuete](http://i.imgur.com/N08wp09.jpg)

Die Chipstuete zeichnet sich durch folgende Charakteristika aus.
- Die Chipstuete besteht aus einem Verbundstoff
- Die Chipstuete reflektiert besonders gerne Licht in einem Wellenlaengenbereich von 560-600nm
- Die Chipstuete ist nicht mit dem auf ihr ausgeschriebenen Inhalt bestueckt.

Die Chipstuete wird bei LAT:48.35757,LON:10.88725 vermutet.
Ihr Bestimmungsort ist der bei LAT:48.35787,LON:10.88755 alloziierte Muelleimer.

Sehen Sie im Folgenden ein Symbolbild fuer einen Mülleimer:

![Muelleimer](http://www.witze-blogger.de/wp-content/uploads/2012/03/muelleimer-zunge.jpg)

**Comment by waaaaargh** [comment]
Problem trat erneut auf. 
![img_20150202_173311](https://cloud.githubusercontent.com/assets/243367/6004642/7ef0e1dc-ab04-11e4-8ce8-710d07c37382.jpg)

**Comment by waaaaargh** [comment]
Ich habe das Problem in der Zwischenzeit behoben, aber da es ein wiederkehrendes Problem ist, sollten wir über seine Ursachen diskutieren.

**Comment by devkral** [comment]
sorry, meine Schuld. Hab um 6 Uhr morgens, vergessen wegzuräumen.

**Comment by lukasepple** [comment]
Muss der Chronist sich in der Pflicht fühlen?

**Comment by moba** [comment]
Uh oh. Müssen wir wirklich über Ursachen diskutieren? Ist es schon so weit? Ich wünsche mir Chipstüten. Vielleicht sind wir uns aber ja einig dass wir eigentlich über verbesserte Prozesse und dadurch automatisch bessere Resultate sprechen wollen, und nicht Ursachen diskutieren müssen.

**Comment by momo-aux** [comment]
Vielleicht sollten wir auch einfach Chipstüten in den Räumlichkeiten verbieten und an der Tür alle vor dem Eintritt in das OpenLab nach Chipstüten durchsuchen, damit wäre das Problem nachhaltig gelöst und die Sicherheit für alle beteiligten erhöht sich dadurch dramatisch. Man könnte auch abschließbare Fächer auf dem Vordach installieren um Besuchern zu ermöglichen ihre Chipstüten vor dem Betreten des Labors auszulagern. Ich stelle mir dann kleine Arbeitsgruppen vor, die gemeinsam das Labor verlassen um aus Chipstüten zu essen. Das trägt doch dann auch zur Geselligkeit bei (ähnlich der grüppchenbildung bei Rauchern). Vielleicht sollten wir einen Sicherheitsbeauftragten wählen, der innerhalb der Räumlichkeiten das Recht hat jeden, jederzeit nach Chipstüten zu durchsuchen. Haltet ihr das nicht für sinnvoll? Ich bin mir sicher es gibt noch andere Lösungen aber Sicherheitstechnisch betrachtet halte ich das für durchaus angebracht.

**Comment by devkral** [comment]
ich bin für eine soziale Lösung: jeder der eine leere Chipstüte liegen lässt, muss 2 Volle mitbringen.

**Comment by waaaaargh** [comment]
@devkral :+1:

**Comment by Profpatsch** [comment]
@devkral +1

**Comment by grindhold** [comment]
Ich fordere einen Guacamole-Koeffizienten.
Der Guacamole-Koeffizient ist die Zeit in vergangenen Sekunden seit dem Auffinden einer Chipstuete und der Bereitstellung der vollen Ersatzchipstueten. Wenn er ermittelt wurde, wird er mit der Guacamole-Konstante (0.1 g/Chip) Multipliziert. Das Ergebnis wird widerum auf die Anzahl der neu angelieferten Chips aufmultipliziert. Man erhaelt die Masse, der Guacamole, die der Delinquent zusaetzlich zu den Chipstueten anliefern muss.

**Comment by robinkaranu** [comment]
Nach kurzer Rücksprache mit Josef, stelle ich die Frage in den Raum, ob wir dieses Thema nicht nochmal von Anfang an aufarbeiten sollten um endlich eine auch mit dem Vereinsrecht optimal abgestimmte Lösung zu finden.

**Comment by Phantom2342** [comment]
Ja, gute Idee. Ich schlage vor, die Satzung wie folgt zu erweitern:

§ Chipstüten-Steuer

(1) Auf jedes Hantieren mit einer Chipstüte in den Laborräumen oder das Essen aus einer solchen wird eine Abgabe gemäß Gebührenordnung erhoben. (2) Die Abgabe, im Folgenden auch Chipstütenbenutzungsbeitrag genannt, ist innerhalb einer 14-Tages-Frist an die OpenLab Augsburg GmbH zu entrichten. (3) Die Einnahmen aus dem Chipstütenbenutzungsbeitrag können dazu verwendet werden, die beim Hantieren mit Chipstüten oder dem Essen aus einer solchen unweigerlich abfallenden Chipskrümel (bayr. Bresel) zu entsorgen. Jedoch ist darauf zu achten, dass die hierdurch entstehenden Ausgaben die Einnahmen nicht übersteigen. (4) Um Anreize zum Hantieren mit Chipstüten oder dem Essen aus einer solchen zu setzen und damit den Gewinn der OpenLab Augsburg GmbH weiter zu erhöhen, ist stets für ausreichenden Nachschub an Chipstüten im Getränke- und Snackautomaten zu sorgen und zu einem angemessenen Preis zu verkaufen.

**Comment by craftamap** [comment]
@Phantom2342 leider kann ich das so nicht in die Satzung aufnehmen - kannst du bitte einen Antrag auf Erweiterung der Satzung für die nächste Mitgliederversammlung in der Sternzeit 48038,5 per Chipkrümel zukommen lassen?
